controller/internal/routes/identity_router.go

```diff
@@ -181,7 +182,8 @@ func (r *IdentityRouter) Update(ae *env.AppEnv, rc *response.RequestContext, par
 
 func (r *IdentityRouter) Patch(ae *env.AppEnv, rc *response.RequestContext, params identity.PatchIdentityParams) {
 	Patch(rc, func(id string, fields JsonFields) error {
```

Comment:
serviceHostingCost and serviceHostingPrecedences weren't being treated as maps, so patch wasn't seeing the name properly and update would fail

Reply:
:+1: